plugin/lsp.vim

```diff
@@ -80,30 +80,30 @@ command! LspNextDiagnostic call lsp#ui#vim#diagnostics#next_diagnostic()
 command! LspPreviousDiagnostic call lsp#ui#vim#diagnostics#previous_diagnostic()
 command! LspReferences call lsp#ui#vim#references()
 command! LspRename call lsp#ui#vim#rename()
-command! LspTypeDefinition call lsp#ui#vim#type_definition(0)
+command! -bang -nargs=? -complete=customlist,lsp#ui#vim#open_where LspTypeDefinition call lsp#ui#vim#type_definition('<args>')
 command! LspTypeHierarchy call lsp#ui#vim#type_hierarchy()
-command! LspPeekTypeDefinition call lsp#ui#vim#type_definition(1)
+command! LspPeekTypeDefinition call lsp#ui#vim#type_definition('preview')
 command! LspWorkspaceSymbol call lsp#ui#vim#workspace_symbol()
 command! -range LspDocumentFormat call lsp#ui#vim#document_format()
 command! -range LspDocumentFormatSync call lsp#ui#vim#document_format_sync()
 command! -range LspDocumentRangeFormat call lsp#ui#vim#document_range_format()
 command! -range LspDocumentRangeFormatSync call lsp#ui#vim#document_range_format_sync()
-command! LspImplementation call lsp#ui#vim#implementation(0)
-command! LspPeekImplementation call lsp#ui#vim#implementation(1)
+command! LspImplementation call lsp#ui#vim#implementation('default')
+command! -bang -nargs=? -complete=customlist,lsp#ui#vim#open_where LspImplementation call lsp#ui#vim#implementation('<args>')
+command! LspPeekImplementation call lsp#ui#vim#implementation('preview')
 command! -nargs=0 LspStatus call lsp#print_server_status()
 command! LspNextReference call lsp#ui#vim#references#jump(+1)
 command! LspPreviousReference call lsp#ui#vim#references#jump(-1)
 command! -nargs=? -complete=customlist,lsp#server_complete LspStopServer call lsp#ui#vim#stop_server(<f-args>)
 command! -nargs=? -complete=customlist,lsp#utils#empty_complete LspSignatureHelp call lsp#ui#vim#signature_help#get_signature_help_under_cursor()
-command! LspDocumentFold call lsp#ui#vim#folding#fold(0)
-command! LspDocumentFoldSync call lsp#ui#vim#folding#fold(1)
+command! LspDocumentFold call lsp#ui#vim#folding#fold('default')
```

Comment:
The parameter refers to sync/async, so you shouldn't change this one :smile:

Reply:
Oops. Did this with regex 😛